### ISSUE TITLE
How to add image captions and alt tags?

### ISSUE THREAD

**Issue by eddively** [GitbookIO/gitbook#1324]
In order to be compliant with WCAG (all public organizations really need to be compliant here), and as someone who is proposing using GitBooks to create written works for government-owned agencies, these are important features. Am I missing something? These seem like standard embedded image features. Loving GitBook, but wondering about this issue.

**Comment by nagim** [comment]
If you're using markdown, it looks like this:
```markdown
![alt text](image path "Title Text ")
```

In asciidoc:
``` asciidoc
image:image path[alt text, title="Title Text"]
```

Hope that helps.

Cheers,
Imre

**Comment by eddively** [comment]
Thanks nagim! This plus the "image-captions" plugin does the trick!
